src/mongosh/data/enumerators.json

```diff
@@ -1,93 +1,114 @@
-            "Weekly": "Meetings are once every week",
-            "Twice a Week": "Meetings are twice every week",
-            "Daily": "Meetings are every day"
-        }
-    }, 
-    "partner": {
-        "status": {
-            "Active":"An active partner",
-            "Inactive":"A partner that has discontinued relationships",
-            "Archived":"Soft Delete Indicator"
-        }
-    },
-    "topic": {
-        "status": {
-            "Active":"",
-            "Inactive":"",
-            "Archived":""
-        },
-        "domain": {
-            "Architecture":"",
-            "Automated Testing":"",
-            "Clean Code":"",
-            "Computing":"",
-            "Craftsmanship":"",
-            "Data":"",
-            "Human Experience":"",
-            "Infrastructure":"", 
-            "Internet":"",
-            "Monitoring":"", 
-            "Operations":"", 
-            "Product Life Cycle":"",
-            "Programming Languages":"",
-            "Security":"", 
-            "Tooling":""
+        "skills": {
+            "status": {
+                "Active": "An active skill",
+                "Archived": "Soft Delete Indicator"
+            },
+            "mastery": {
+                "Candidate":"An Apprentice Candidate learning a full-stack",
+                "Apprentice": "A software engineer gaining broad skills",
+                "Resident": "A well-rounded software engineer with a grasp of technologies across all ITIL/ITSM domains and an area of Specialization",
+                "Master": "A senior software engineer recognized as a leader within their specialty",
+                "Distinguished": "A senior software engineer that has been recognized by their peers as a distinguished contributor"
+            },
+            "speciality": {
+                "Generalist": "General skills not associated with a specific specialty",
+                "UI/UX Engineer": "UI/UX software engineer",
+                "API Engineer": "API Engineer",
+                "Data Engineer": "Data Engineer",
+                "SRE Engineer": "Site Reliability Engineer"
+            }
         },
-        "resourceType": {            
-            "Article":"",
-            "Book":"",
-            "Certification Exam":"",
-            "Course":"",
-            "Design System":"",
-            "Document":"",
-            "Exercise":"",
-            "EngineerKit Resource":"",
-            "Film":"",
-            "Game":"",
-            "Guide":"",
-            "Image":"",
-            "Official Guide":"",
-            "Official Training":"",
-            "Resource":"",
-            "Tool":"", 
-            "Thread":"",
-            "Tutorial":"",
-            "Video":"",
-            "Video Series":"",
-            "Website":""
+        "resources": {
+            "status": {
+                "Active": "An active resource",
+                "Archived": "Soft Delete Indicator"
+            },
+            "tags": {
```

Comment:
Consider adding a couple for accessibility purposes like "subtitles" OR "transcript" for audio content